src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -474,28 +510,28 @@ function hideAllNewQuestionForms() {
 function prepareQuestionForm(type) {
     switch (type) {
     case 'TEXT':
-        $('#questionTypeHeader').append(FEEDBACK_QUESTION_TYPENAME_TEXT);
+        $('#questionTypeHeader').html(FEEDBACK_QUESTION_TYPENAME_TEXT);
         
         hideAllNewQuestionForms();
         break;
     case 'MCQ':
         $('#' + FEEDBACK_QUESTION_NUMBEROFCHOICECREATED + '--1').val(2);
-        $('#questionTypeHeader').append(FEEDBACK_QUESTION_TYPENAME_MCQ);
+        $('#questionTypeHeader').html(FEEDBACK_QUESTION_TYPENAME_MCQ);
```

Comment:
The `#questionTypeHeader` is a text label showing the question type for the new question that is added by clicking on the new question button.

Previously, initializing the label by appending works because users cannot add a question more than once. (If they cancel adding a question, the page will reload.)

Now, users can add a new question, click cancel, then add another question. This will result in the question type being appended twice.